src/main/java/com/iemr/admin/service/telemedicine/VideoConsultationAPIServiceImpl.java

```diff
@@ -32,34 +32,33 @@
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
 import com.iemr.admin.utils.config.ConfigProperties;
-import com.iemr.admin.utils.exception.SwyMedException;
+import com.iemr.admin.utils.exception.VideoConsultationException;
 import com.iemr.admin.utils.http.HttpUtils;
 import com.iemr.admin.utils.mapper.InputMapper;
 import com.iemr.admin.utils.response.OutputResponse;
 
 @Service
-public class SwymedAPIServiceImpl implements SwymedAPIInter {
+public class VideoConsultationAPIServiceImpl implements VideoConsultationAPIInter {
 
 	Logger logger = LoggerFactory.getLogger(this.getClass().getName());
 	private static HttpUtils httpUtils = new HttpUtils();
-	private String swymedauth = ConfigProperties.getPropertyByName("swymed-apikey");
-	private String swymed_base_url = ConfigProperties.getPropertyByName("swymed-base-url");
-	private String swymed_createuser = ConfigProperties.getPropertyByName("swymed-createuser-url");
-	private String swymed_edituser = ConfigProperties.getPropertyByName("swymed-edituser-url");
+	private String videoConsultationauth = ConfigProperties.getPropertyByName("videoConsultation-apikey");
+	private String videoConsultation_base_url = ConfigProperties.getPropertyByName("videoConsultation-base-url");
```

Comment:
@Vidyaaa24 Change the variable name videoConsultation_base_url to videoConsultationBaseUrl